apps/ledger-live-mobile/src/components/Carousel/Slide.tsx

```diff
@@ -39,16 +40,19 @@ const Slide = ({
 }: SlideProps) => {
   const { t } = useTranslation();
   const { navigate } = useNavigation();
+  const currentScreen = useCurrentRouteName();
   const onClick = useCallback(() => {
-    track("Portfolio Recommended OpenUrl", {
+    track("banner_clicked", {
+      banner: "Discover Web3",
```

Comment:
I'm not sure about this one, why are we passing `Discover Web3` for all the slides ? Shouldn't we pass the title instead ?

Reply:
![image](https://user-images.githubusercontent.com/8106590/185379615-7e47f9fb-c8bf-4825-b7ac-56894384f907.png)
But yes title of card clicked should be the one to go. I was ust using the spec here.

Reply:
changed as banner name